apm-sniffer/optional-plugins/trace-ignore-plugin/pom.xml

```diff
@@ -41,5 +42,11 @@
             <version>${ststem-rules.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
```

Comment:
We can't accept Spring core, sorry. Because we have to include all codes inside the agent plugin.